toolchain/check/operator.cpp

```diff
@@ -0,0 +1,145 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "toolchain/check/operator.h"
+
+#include "toolchain/check/call.h"
+#include "toolchain/check/context.h"
+#include "toolchain/check/member_access.h"
+#include "toolchain/sem_ir/ids.h"
+#include "toolchain/sem_ir/typed_insts.h"
+
+namespace Carbon::Check {
+
+// Returns the scope of the Carbon package, or Invalid if it's not found.
```

Comment:
Why "Carbon" versus "Core"? Wasn't "Core" the outcome of https://github.com/carbon-language/carbon-lang/issues/2113#issuecomment-1301589247?